CharacterCreation/03_Feats.txt

```diff
@@ -689,113 +696,181 @@ now working with. This feat can only be taken once per weapon.
+36:	+2 to Accuracy rolls at all ranges while firing automatically (not burst).
+48:	+15m to range R3 while firing semi-automatically.
+60:	+10% Base damage. While suppressing an enemy, if you are shot at, after 
+	the first Action of fire you may spend a reaction to duck behind cover 
+	within 5m. This ends your suppression.
+
+== Shotgun Expertise  ==
+
+0:	+5m to range R1. If this would cause R1 to be greater than R2, 
+	also set R2 equal to the new R1.
+12:	May use Weapons - Shotgun (Dex) in place of Acrobatics for checks to execute 
+	a physical maneuver following violent entry of a room.
+24:	If you succeed a reload check by at least 20, reduce the time taken to 
+	reload by one Action, stacking with other reductions (such as from 
+	critical success), to a minimum of one Action.
+36:	Negate the hip-firing Accuracy penalty against enemies within R1 while 
+	firing spreading shot.		
+48:	+10m to range R2. If this would cause R2 to be greater than R3, 
+	also set R3 equal to the new R3.
+60:	+10% Base damage. If you hit armor with a higher APL than your shotgun, 
+	you deal half damage to its AS instead of no damage (and full damage 
+	to health if you deplete the AS).
+
+== Carbine Expertise ==
+
+0:	+10m to range R2. If this would cause R2 to be greater than R3, 
+	also increase R3 to match the new R2.
+12:	May use Weapons - Carbine (Dex) in place of Acrobatics on checks to maintain 
+	balance and negate combat penalties associated with a moving vehicle or mount 
+	that you are on board.
+24:	+25 Weapon Reflex while onboard a moving vehicle or mount and for 3 rounds 
+	after disembarking. +10 Weapon Reflex at other times.
+36:	+1 Accuracy to ranges R1 and R2 while firing while benefiting from cover 
+	of any kind against your target.
+48:	+10m to range R1, +5m to range R3
+60:	+10% Base damage. For up to one Action per round you may fire your carbine one-handed with  
+	a -2 Accuracy penalty and double any burst or automatic fire Accuracy penalty. Any foregrips or 
+	other attachments requiring the second hand do not provide their benefits during the 
+	Action, but your offhand is free to do something else. If you have taken aim at a target, 
+	you must succeed on a DC 100 Weapons - Carbine check to maintain that aim during and 
+	through the Action.
+
+== Assault/Battle Rifle Expertise ==
+
+0:	+10m to range R2. If this would cause R2 to be greater than R3, 
+	also increase R3 to match the new R2.
+12:	May use Weapons - Assault/Battle Rifle in place of Spotting when 
+	looking through your rifle's scope (warning, takes longer than 
+	wider field-of-view optics).
+24:	Halve the jam chance (applies after additive modifiers, round 
+	down, may be zero) of your assault/battle rifle for the first 3 
+	rounds of its use per combat, once per short rest (i.e. maintenance 
+	& cleaning oportunity).
+36:	+1 Accuracy while both you and your target are benefiting from 
+	cover bonuses against each other.

```

Comment:
Clever! That way you use the cover bonuses to prevent running into the hard cap, and you strongly encourage new players (who are more likely to be playing soldier) to always fight from cover! That's pretty sweet, dude!